RadarSDK/RadarSettings.m

```diff
@@ -239,6 +240,15 @@ + (NSString *)verifiedHost {
     return verifiedHost ? verifiedHost : kDefaultVerifiedHost;
 }
 
++ (BOOL)userDebug {
```

Comment:
We also need the iOS analog of [this change on Android](https://github.com/radarlabs/radar-sdk-android/commit/7f7ffbe800ab08662ea39a135fc88408a362453a#diff-e17c923825657ab70c94cca4e224e5a6c65d35a8fc2bfecb7e42dc9527a89d8bR256-R257) in the `logLevel` function